linode/nb/framework_resource.go

```diff
@@ -195,6 +214,8 @@ func (r *Resource) Update(
 			return
 		}
 
+		tflog.Trace(ctx, "client.UpdateNodeBalancer(...)")
```

Comment:
Since it's an important API call, can we use debug and also log the `updateOpts`?